### PR TITLE
353 Add edit programme details confirmation page

### PR DESCRIPTION
**Pull request by SamJamCul**
### Context
https://trello.com/c/MAQqgkeA/353-s-confirm-trainee-programme-details-page

### Changes proposed in this pull request
* Added a confirmation page and summary card after the edit programme details section

### Guidance to review

image

### CODE REVIEW

**Comment by SamJamCul** [comment]
[[S] Confirm Trainee Programme Details Page](https://trello.com/c/MAQqgkeA/353-s-confirm-trainee-programme-details-page)

File: spec/features/trainees/edit_programme_details_spec.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
     when_i_visit_the_programme_details_page
     and_i_enter_valid_parameters
     and_i_submit_the_form
+    and_confirm_my_details
```
Comment (defong) [inline spec/features/trainees/edit_programme_details_spec.rb:9]:
```suggestion
    and_i_confirm_my_details
```

File: spec/features/trainees/edit_programme_details_spec.rb [hunk 2]
```diff
@@ -67,6 +68,12 @@ def and_the_programme_details_are_updated
     end
   end
 
+  def and_confirm_my_details
```
Comment (defong) [inline spec/features/trainees/edit_programme_details_spec.rb:71]:
```suggestion
  def and_i_confirm_my_details
```

**Review by defong — CHANGES_REQUESTED** [review]
> ### Guidance to review
> 
> image

image

These two looks like it needs addressing
- [ ] title
- [x] date format

File: spec/features/trainees/edit_programme_details_spec.rb (hunk 2 above)
Comment (SamJamCul) [reply spec/features/trainees/edit_programme_details_spec.rb]: Done

File: spec/features/trainees/edit_programme_details_spec.rb (hunk 1 above)
Comment (SamJamCul) [reply spec/features/trainees/edit_programme_details_spec.rb]: done

**Comment by SamJamCul** [comment]
@defong I checked other confirmation cards, and none of them have this dynamic titling based on the Assessment Only route thing - I think this might be something we have to add later, or as another card at least?

**Comment by defong** [comment]
> @defong I checked other confirmation cards, and none of them have this dynamic titling based on the Assessment Only route thing - I think this might be something we have to add later, or as another card at least?

yeah... I was comparing it with prototype... I guess we can card it for later

**Review by defong — APPROVED** [review]

**Review by darokel — APPROVED** [review]

**Pull request merged**